cmd/snap/cmd_download.go

```diff
@@ -103,6 +107,9 @@ func (x *cmdDownload) Execute(args []string) error {
 		if x.Channel != "" {
 			return fmt.Errorf(i18n.G("cannot specify both channel and revision"))
 		}
+		if x.CohortKey != "" {
```

Comment:
Can we start growing some cli side unit tests for this? Just basic arg handling and errors for starters would be fine (a followup is ok)

Reply:
I'll add it now. Can't do more than check errors though.

Reply:
added

Reply:
Thanks!